.github/workflows/cxx_interop.yml

```diff
@@ -0,0 +1,38 @@
+name: Cxx interop
+
+on:
+  workflow_call:
+    inputs:
+        linux_5_8_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.8 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_9_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.9 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_10_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.10 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_6_0_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly 6.0 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_main_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly main Swift version matrix job. Defaults to true."
+            default: true
+ 
+jobs:
+    cxx-interop:
+        name: Cxx interop
```

Comment:
Nit: Any reason to not spell this as "C++" when we have a string label?

Reply:
Kinda stayed consistent with the script name and the flag that's set in the `Package.swift`. Also AFAIK all of C++/Cpp/Cxx is valid to use.

Reply:
Sure, they're all valid and I'm fine with keeping it. I always assumed Cxx and Cpp were there for when we couldn't use non-ascii characters, like in command line flags and file extensions.